snap/snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: ngs # you probably want to 'snapcraft register <name>'
+base: core18 # the base snap is the execution environment for this snap
+version: '0.2.9' # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
Sounds like modifying this should be documented at https://github.com/ngs-lang/ngs/blob/master/release.md

Reply:
Added the instructions :smile: